src/Employer/Employer.Web/Orchestrators/Part2/AboutEmployerOrchestrator.cs

```diff
@@ -30,7 +31,7 @@ public async Task<AboutEmployerViewModel> GetAboutEmployerViewModelAsync(Vacancy
             var vm = new AboutEmployerViewModel
             {
                 Title = vacancy.Title,
-                EmployerDescription = vacancy.EmployerDescription,
+                EmployerDescription = await GetEmployerDescriptionAsync(vacancy),
```

Comment:
Just a thought. I wonder if we could/should move these Employer Description manipulations into `Utility.GetAuthorisedVacancyForEditAsync`. Maybe use the vacancy state and/or route to determine if the vacancy description should be changed or not.

Dunno, may be a terrible idea!

Reply:
Hmmmm...my choice would be to leave it where it is. I think the Utility class should be left to be relatively simple otherwise we may be in danger of it becoming a bit of a dumping ground.